test/hotspot/jtreg/compiler/codecache/stress/CodeCacheStressRunner.java

```diff
@@ -27,17 +27,21 @@
 import jdk.test.lib.Utils;
 
 public class CodeCacheStressRunner {
+    private static long end_time;
     private final Runnable action;
+
     public CodeCacheStressRunner(Runnable action) {
         this.action = action;
     }
 
     protected final void runTest() {
         Helper.startInfiniteLoopThread(action);
         try {
-            // adjust timeout and substract vm init and exit time
+            // Adjust timeout and substract vm init and exit time
+            // Runs with fixed workload that is capped by the TimeLimitedRunner
+            end_time = System.nanoTime() + 60 * 1000 * 1000 * 1000L;
             long timeout = Utils.adjustTimeout(Utils.DEFAULT_TEST_TIMEOUT);
-            timeout *= 0.8;
+            timeout *= 0.75;
             new TimeLimitedRunner(timeout, 2.0d, this::test).call();
```

Comment:
why do you need `end_time`? won't it be enough to just set `timeout` to 60_000?

Reply:
I had a much more complex version at first. I realize that now it is equivalent to just setting 60 secs. 

I have fixed in in the update.